test/suites/E2E/test_environment_precheck.py

```diff
@@ -0,0 +1,238 @@
+import os
+from pathlib import Path
+from typing import Any, Dict
+
+import pytest
+import yaml
+from common.capture_utils import export_vars
+from common.envPreCheck.run_env_preCheck import (
+    LOCAL_SSH_KEY,
+    MASTER_IP,
+    WORKER_IP,
+    check_env_exists,
+    run_bandwidth_check,
+    run_check_hccn_ping,
+    run_check_model_weight,
+    run_check_nvidia_ping,
+    run_device_status_check,
+    run_hccn_tls_check,
+    run_set_ssh_login,
+)
+
+
+# ========= Environment Setup =========
+@pytest.mark.stage(0)
+@pytest.mark.feature("test_ssh_login")
+@export_vars
```

Comment:
this decorator is only for internal usage and we may not need to update external database while doing precheck things